src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -171,6 +171,10 @@ public function editAction($customerId, Request $request)
             $result = $customerFormHandler->handleFor((int) $customerId, $customerForm);
 
             if (null !== $result->getIdentifiableObjectId()) {
+                if ($request->query->has('back')) {
```

Comment:
@matks wdyt if we had some helper method to do redirect,but also check for `back` parameters. This is behavior that comes from legacy.

It could be something like `$this->redirectWithBack('admin_customers_index')` (yup, name should be improved), which would check if there is `back` parameter in `query` before redirecting to given route.

Reply:
Thinking it's a little bit overkill because back param depends on which page we are not generic.

Reply:
Either way, we are missing `back` "feature" in new pages. :/

Reply:
Sad moment :disappointed:

Reply:
I like @sarjon idea and it seems it works for me as long as `back` contains the whole URL, like in legacy behavior. @PierreRambaud can you explain what you mean by `which page we are not generic` ?

Reply:
https://github.com/PrestaShop/PrestaShop/pull/12971/files#diff-30f66bdfdec458e744e8d9cd497f1345L633 Not sure all back urls are like this one :)

Reply:
Well, we can make this `back` redirection a bit smart and append the domain name if needed. Being able to handle `https://www.a.com/index.php`, `www.a.com/index.php` and `/index.php` is not very hard

Reply:
Created issue https://github.com/PrestaShop/PrestaShop/issues/12990 to handle this matter